README.md

```diff
@@ -765,11 +765,17 @@ Normally Output Bindings require two things :
 
 If either of these are false then an error will be thrown.
```

Comment:
`Normally if either of these are false then an error will be thrown. Below are the situations in which this is not the case :`

(and then probably reword the first sentence in each section to read a bit better)